frontend/src/layout.ts

```diff
@@ -55,19 +73,44 @@ export class StatefulLayout {
     }
   }
 
-  nodeColor(eventType: string, hue?: number) {
-    if (!this.colors.has(eventType)) {
-      const hash = [...eventType].reduce(
-        (acc, char) => char.charCodeAt(0) + ((acc << 1) - acc),
-        2
-      )
-      const color = `hsl(${hash % (hue ? hue : 360)},50%,50%)`
-      this.colors.set(eventType, color)
-    }
-    return this.colors.get(eventType)
+  nodeColor(eventType: string) {
+    // if (!this.colors.has(eventType)) {
+    //   const hash = [...eventType].reduce(
+    //     (acc, char) => char.charCodeAt(0) + ((acc << 1) - acc),
+    //     2
+    //   )
+    //   const color = `hsl(${hash % (hue ? hue : 360)},50%,50%)`
+    //   this.colors.set(eventType, color)
+    return this.themeMap.get(eventType).Color
```

Comment:
Default case should be applied here (`defaultColor`) for the others as well

Reply:
I think the G6 lib come with default shape and color for the node and also the config.json file have all eiffle event coverd, so I did not implement that.